types/chat.ts

```diff
@@ -155,10 +159,24 @@ export interface QuickActions {
 }
 
 export interface TabData {
-    placeholderText?: string
+    title?: string
+    tabHeader?: TabHeader
+    promptInput?: PromptInput
+    compactMode?: boolean
     messages: ChatMessage[]
 }
 
+export interface TabHeader {
+    icon?: IconType
+    title?: string
+    description?: string
+}
+
+export interface PromptInput {
```

Comment:
Why don't these have docstrings that give insight into how they should be used? Is that documented somewhere else? If not, can we make that a habit for all runtimes PRs?